packages/app/scripts/deploy.js

```diff
@@ -0,0 +1,31 @@
+const fetch = require('whatwg-fetch');
```

Comment:
> This project **doesn't work under Node.js environments**. It's meant for web
  browsers only. You should ensure that your application doesn't try to package
  and run this on the server.

https://github.com/github/fetch/blame/master/README.md#L46-L48